crates/stages/src/pipeline/mod.rs

```diff
@@ -72,7 +72,8 @@ use state::*;
 /// pipeline will unwind the stages in reverse order of execution. It is also possible to
 /// request an unwind manually (see [Pipeline::unwind]).
 pub struct Pipeline<DB: Database, U: SyncStateUpdater> {
-    stages: Vec<QueuedStage<DB>>,
+    /// The stages that are part of the pipeline
+    pub stages: Vec<QueuedStage<DB>>,
```

Comment:
recommend making this private as well and accessing via function, I feel like we'll end up changing this a bit.